x-pack/plugins/fleet/public/applications/fleet/sections/settings/components/edit_output_flyout/encryption_key_required_callout.tsx

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { EuiCallOut, EuiLink, EuiCode } from '@elastic/eui';
+
+import { useStartServices } from '../../../../hooks';
+
+export const EncryptionKeyRequiredCallout: React.FunctionComponent = () => {
+  const { docLinks } = useStartServices();
+  return (
+    <EuiCallOut
+      iconType="alert"
+      color="warning"
+      title={
+        <FormattedMessage
+          id="xpack.fleet.encryptionKeyRequired.calloutTitle"
+          defaultMessage="Set an encryption key to enable logstash output"
+        />
+      }
+    >
+      <FormattedMessage
+        id="xpack.fleet.encryptionKeyRequired.calloutDescription"
+        defaultMessage="To configure logstash output, set a value of {key} in your {file} file. {link}"
```

Comment:
nit: I wonder if we should use different wording here since not all environments use kibana.yml (eg. docker users use env vars often).

Reply:
yes we probably can update this, (I think it can come as a follow up PR as merging this one will unlock testing the logstash output) the doc do not specify the docker env var variables either maybe it could be fixed here.